### PR TITLE
updating to 1.0.6 with bug fix for GT merging

### PR DESCRIPTION
**Pull request by apaul7**
in v1.0.5 there was an issue when merging vcfs that contained fields but no GT field. this PR updates to the latest version of survivor and contains the bug fix. 

Had to use a forked copy of the SURVIVOR repo to get the bug fix as survivor released 1.0.6 a few days before fixing this bug.

### CODE REVIEW

**Comment by apaul7** [comment]
I'm trying to keep the master branch without a Dockerfile to make sure that dockerhub does not create an image of it and to make sure people specify exactly what version they want when using the image instead of `latest`. What's the best way for making a PR into a to be created branch? 
I ended up manually making a branch in this repo then making a PR to that just created branch from my forked repo. Tried to make the PR merge directly to a new branch in the main repo but I was unable to create the PR

**Comment by tmooney** [comment]
The easiest approach is to make a PR against master with each new release and then branch from that after it's merged.  If that's acceptable you could remove the rule on Dockerhub that builds `latest` from `master`.

Otherwise, what you've done is probably the next best solution.  It's harder to compare to the previous Dockerfile, but it avoids touching `master`.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:xenial
+MAINTAINER Alexander Paul <alex.paul@wustl.edu>
+
+LABEL \
+  version="1.0.5" \
```
Comment (tmooney) [inline Dockerfile]: Need to bump the version here:
```suggestion
  version="1.0.6" \
```

**Review by tmooney — COMMENTED** [review]
Github thinks these files are missing their final newlines.

(Should the version of this one be something beyond just `1.0.6` if it includes extra changes that aren't in the release?)

**Comment by apaul7** [comment]
Yes that part of the Dockerfiles should be updated. But the more I think about it I don't know the best answer for the versioning. The code is `v1.0.6` with the bug fix for actually checking if the `GT` field is in the vcfs it's trying to merge. But in the output vcf SURVIVOR says that it is running `v1.0.7` because he upped the version in the codebase. I'm hesitant to say it's `v1.0.7` because what if more things are added before it's officially released but I also don't want it to misrepresent what the results will be.

I guess since it's my fork I could replace the version in the output vcf?

**Comment by apaul7** [comment]
@tmooney could you take one more look at this? I've updated my fork of survivor to output version `1.0.6.1`

**Review by tmooney — APPROVED** [review]
+1; should the branch this targets also be `1.0.6.1`?

**Comment by apaul7** [comment]
yup! merging

**Pull request merged**